docs/doxygen/mainpage.md

```diff
@@ -11,137 +11,28 @@ The Fw package provides the core classes, components, and ports to support F´.
 for the core framework operation. In addition, these include the base classes on which components
 are built.  Finally, of specific interest is Fw::Types providing the cored types in the system.
 
-\subpage FwBufferSerializableBufferGetBufferSend
```

Comment:
Why?

Reply:
Read the above comment.  Although I agree about maintainability, I want to make sure the SDDs are appearing in the doxygen too.

Reply:
The SDD's are listed in the sidebar - is that sufficient?

![image](https://github.com/nasa/fprime/assets/49786685/c63413da-7420-4340-8f81-37e3b4b6a16f)